src/sentry/rules/conditions/high_priority_issue.py

```diff
@@ -33,6 +33,15 @@ def passes(self, event: GroupEvent, state: EventState) -> bool:
         if not has_high_priority_issue_alerts(self.project):
             return False
 
+        if not self.rule:
+            return state.is_new
```

Comment:
Is this just to placate the type checker? I think that in practice, it's not actually possible that the rule is None

Reply:
yup, just the typing